Page breaks

ChefAndy opened the pull request:
As usual, this is radically different from the way I started to do it but it's a lot simpler and seems to work well. Had what seemed to be some problems but I was mostly being dumb and they were easy to solve after I worked on other things for a few days. woot.

ChefAndy committed:
changed HTML generation to handle styled XML

ChefAndy committed:
change html PB to <br> and refactor merge_alto_style

ChefAndy committed:
Better test fixed a problem that test found

In the end, this ended up being simpler than I had anticipated.

modified:   scripts/merge_alto_style.py— checks to make sure the
					element we're preceeding
					the <pagebreak> with is
					for the first text block
					in the alto file.
modified:   scripts/tests/test_helpers.py— new, better test

ChefAndy committed:
Merge branch 'develop' into page_breaks

jcushman commented on capstone/scripts/generate_case_html.py:
```diff
@@ -41,6 +42,16 @@ def generate_html(case_xml, tag_map=tag_map):
+        element_text_copy = element.text
+
+        if element_text_copy is None and tag != 'pagebreak':
+            if len(element.getchildren()) == 0:
```
Remove debugging line

jcushman reviewed (changes requested):
Just one tweak ...

ChefAndy committed:
Update generate_case_html.py

At Jack's behest, I rolled back the number of debug statements in my production code to zero— a common preference among people who prefer code that works in all foreseeable circumstances.
